iml-system-test-utils/src/lib.rs

```diff
@@ -732,20 +711,27 @@ pub async fn test_stratagem_taskqueue(config: Config) -> Result<Config, TestErro
     .await?;
 
     // Create Task
-    let cmd = r#"iml debugapi post task -"#;
-    let task = br#"{"filesystem": 1, "name": "testfile", "state": "created", "keep_failed": false, "actions": [ "stratagem.warning" ], "single_runner": true, "args": { "report_name": "test-taskfile.txt" } }"#;
-    let mut ssh_child = ssh::ssh_exec_cmd(config.manager_ip, &cmd)
-        .await?
-        .stdin(Stdio::piped())
-        .stdout(Stdio::inherit())
-        .spawn()?;
+    let q = task::create::build(
+        "fs",
+        TaskArgs {
+            name: "testfile".into(),
+            single_runner: true,
+            keep_failed: false,
+            actions: vec!["stratagem.warning".into()],
+            pairs: vec![KeyValue {
+                key: "report_name".into(),
+                value: "test-taskfile.txt".into(),
+            }],
+            needs_cleanup: false,
+        },
+    );
 
-    let ssh_stdin = ssh_child.stdin.as_mut().unwrap();
-    ssh_stdin.write_all(task).await?;
-    ssh_child.wait_with_checked_output().await?;
+    ssh::graphql_call::<_, iml_graphql_queries::Response<task::create::Resp>>(&config, &q).await?;
 
-    // send fid
+    // TODO wait for command to complete
```

Comment:
I thought we had a function to wait for a command to complete. Will that not work here?

Reply:
Not without some work. It currently uses http api calls internally and the current approach here is using SSH.

That said, we should probably just do our calls over HTTP anyway, we already are exposing the port in vagrant.